src/main/scala/com/gu/support/workers/GetRecurringSubscription.scala

```diff
@@ -0,0 +1,54 @@
+package com.gu.support.workers
+
+import cats.implicits._
+import com.gu.support.workers.lambdas.IdentityId
+import com.gu.support.workers.model.AccessScope.{AccessScopeBySessionId, AccessScopeNoRestriction, SessionId}
+import com.gu.support.workers.model.{AccessScope, BillingPeriod}
+import com.gu.zuora.GetAccountForIdentity.DomainAccount
+import com.gu.zuora.GetSubscription.DomainSubscription
+import com.gu.zuora.ZuoraConfig.RatePlanId
+import com.gu.zuora.ZuoraService
+import com.gu.zuora.model.response.RatePlan
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object GetRecurringSubscription {
```

Comment:
this is basically just the def getRecurringSubscription from zuoraService only extended as necessary, and split into some pure functions as I needed to unit test isInAccessScope